de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,179 @@
+                "--enable-muxer=ogg",
+                "--enable-muxer=webm",
+                "--enable-demuxer=avi",
+                "--enable-demuxer=h264",
+                "--enable-demuxer=mov",
+                "--enable-demuxer=mp3",
+                "--enable-demuxer=ogg",
+                "--enable-demuxer=wav",
+                "--enable-parser=h264",
+                "--enable-parser=vorbis",
+                "--enable-encoder=aac",
+                "--enable-encoder=libtheora",
+                "--enable-encoder=libvorbis",
+                "--enable-encoder=libvpx_vp8",
+                "--enable-encoder=libvpx_vp9",
+                "--enable-encoder=libx264",
+                "--enable-encoder=mpeg4",
+                "--enable-decoder=aac",
+                "--enable-decoder=h264",
+                "--enable-decoder=libvorbis",
+                "--enable-decoder=libvpx_vp8",
+                "--enable-decoder=libvpx_vp9",
+                "--enable-decoder=mp3",
+                "--enable-decoder=mpeg4",
+                "--enable-decoder=pcm_s16le",
+                "--enable-decoder=theora"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.ffmpeg.org/releases/ffmpeg-5.0.1.tar.xz",
+                    "sha256": "ef2efae259ce80a240de48ec85ecb062cecca26e4352ffb3fda562c21a93007b"
+                }
+            ],
+            "cleanup": [
+                "/lib/codecs/include",
+                "/lib/codecs/lib/pkgconfig",
+                "/lib/codecs/share"
+            ]
+        },
+        {
+            "name": "bforartists",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 bforartists.sh /app/bin/bforartists",
+                "mv bforartists /app/bforartists",
+                "install -Dm644 /app/bforartists/bforartists.desktop /app/share/applications/$FLATPAK_ID.desktop",
+                "desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -Dm644 /app/bforartists/bforartists.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg",
+                "install -Dm644 /app/bforartists/bforartists-symbolic.svg /app/share/icons/hicolor/symbolic/apps/$FLATPAK_ID-symbolic.svg",
+                "cd /app/bforartists/*.*/datafiles && mv locale /app/share/locale && ln -sf /app/share/locale locale",
+                "install -Dm644 $FLATPAK_ID.appdata.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml"
+            ],
+            "post-install": [
+                "for s in {16,22,24,32,48,64,72,96,128,192,256,512}; do rsvg-convert \"/app/bforartists/bforartists.svg\" -w \"${s}\" -h \"${s}\" -a -f png -o \"$FLATPAK_ID.png\"; [[ ! -f \"/app/share/icons/hicolor/${s}x${s}/apps/$FLATPAK_ID.png\" ]] || continue; install -p -D -m 0644 \"$FLATPAK_ID.png\" -t \"/app/share/icons/hicolor/${s}x${s}/apps/\"; done;"
```

Comment:
if I have a SVG icon you don't need to do this at all.

Reply:
This was taken from an example given in another PR. There doesn't seem to be any documentation for how Flathub wants the icons set up, so please tell me what you suggest.

Reply:
Just SVG is fine, the second best is 512x512 PNG.